### ISSUE TITLE
Interested in proposing a new specification; want to initiate a conversation

### ISSUE THREAD

**Issue by CxRes** [WICG/admin#185]
I am in the process of developing a new notifications protocol called [Per Resource Events](https://cxres.github.io/per-resource-events/protocol). I am currently evaluating various paths to incubate this work. In this regard, I have a number of question for pursuing this specification through WICG. Is there a place where I can informally chat with or (better still) talk to members of WICG for advice?

**Comment by yoavweiss** [comment]
The best way to do that nowadays would be to file an issue on the [Proposals](https://github.com/WICG/proposals) repo.

Otherwise, I think it'd be interesting to expand on the use cases you're hoping the proposal can tackle. One question I had when reading through it - why isn't a [Resource Timing](https://developer.mozilla.org/en-US/docs/Web/API/Performance_API/Resource_timing) [Performance Observer](https://developer.mozilla.org/en-US/docs/Web/API/PerformanceObserver) good enough to get per-resource notifications?

**Comment by CxRes** [comment]
@yoavweiss Thanks for your response. The questions I wanted to be clarified relate to the WICG process and how you might handle things to my specific circumstance. That's why I posted here in the admin repo and wanted to have conversation first. I would be very happy to make a proposal once my questions are addressed. 

To give you a little context to this work, I am an author on [Solid Notification Protocol](https://github.com/solid/notifications). The Solid Notifications Protocol adds notification capabilities to Solid Pods (though it is universal in that it can be used by any other server). [Solid Specification](https://github.com/solid/specification) itself is at a stage of maturity that it will be graduating to W3C WG in fall 2023. 

The protocol that I am proposing is a consequence of a desire to have a simpler protocol, that does not cover every use case of Solid Notifications (SNP can basically accommodate multiple protocols like WebSockets, Webhooks, Streaming HTTP, LDN etc.) but can get people up and running much faster and with lower complexity. There is already an expression of interest in the Solid community to implement my proposal.

Re: Resource Timing/Performance Observers, I am vaguely familiar with these specs, but I thought it had to primarily to do with timing, not changes in representation. I not sure how a user agent would be able to negotiate the content-type of notifications (as well the content-type of diffs for the notification media-types that allow it, See Appendix A). Enlighten me if you have any ideas! Also, PREP works with ordinary fetch, which is its big superpower.

**Comment by marcoscaceres** [comment]
Hmmm... tricky one, because solid is not really a "browser" or "web platform" thing, so it's falling outside the scope of with WICG. I would recommend instead spinning up a "Solid Community Group". 

We have other working groups that do the same thing... like the Immersive Web Working Group has a companion Immersive Web Community Group. That Community Group is used for incubations, while the Working Group does standardization. 

Does that route sound good?

**Comment by marcoscaceres** [comment]
Closing as answered

**Comment by CxRes** [comment]
Perhaps I failed to make this clear, but the specification I am proposing is not tied to Solid and is a general extension to HTTP (and will be relevant to browsers and the web platform in general). It just originates from my work in Solid. Solid already has a CG and soon a WG as well. 

It is for this reason (the generality of PREP) as an HTTP extension (not Solid extension), I had approached WICG for incubation.
